### ISSUE TITLE
Github Package for 2.1.8

### ISSUE THREAD

**Issue by frost-byte** [DevLeoko/AdvancedBan#329]
_What version of AdvancedBan (`/AdvancedBan`) are you using?_
 2.1.7 want to use 2.1.8
 
_What kind of server do you have (`Bungeecord/single server`)?_
 Single
 
_What server version (/version) are you using?_
PaperMC 1.15.2

_Please provide the EXACT steps required to reproduce the problem..._
1. Add dependency to pom.xml of a project
2. Specify version 2.1.8
3. Can also check the packages page in this repo, only shows version 2.1.7

_Any error/log post it through [pastebin.com](http://pastebin.com) and link it here._

_Add any additional information below._

**Comment by DevLeoko** [comment]
GitHub Packages does not seem to be a public maven repo as I thought but is rather meant for internal maven dependency management.
So sorry but AdvancedBan is currently not available through any maven repo and has to be added as a dependency by adding it to the build path.

**Comment by frost-byte** [comment]
Actually, in some sense you are correct. But, in reality all that us needed is a Github account in order to use packages from public Github package repos. 

 I spent some time yesterday troubleshooting this.  It requires the creation of a Personal Access Token and the configuration of your settings.xml for maven.

I created a [wiki](https://github.com/frost-byte/maven-packages/wiki/Using-the-Packages) to describe how to do this and included some links there for reference.
